src/ResourceManager/Sql/Commands.Sql/Replication/Cmdlet/NewAzureSqlDatabaseSecondary.cs

```diff
@@ -44,7 +44,8 @@ public class NewAzureSqlDatabaseSecondary : AzureSqlDatabaseSecondaryCmdletBase
         /// Gets or sets the name of the service objective to assign to the secondary.
         /// </summary>
         [Parameter(Mandatory = false,
-            HelpMessage = "The name of the service objective to assign to the secondary.")]
+            HelpMessage = "The name of the service objective to assign to the secondary." +
+            "Possible ServiceObjectiveName could be S3, P2, GP_Gen4_2. GP_Gen4_2 is new supported service tier.")]
```

Comment:
don't add this.

Reply:
@akromm will fix. same for the above one.